examples/lie_labs_api.py

```diff
@@ -0,0 +1,76 @@
+from typing import cast
+
+import torch
+
+import theseus.labs.lie as lie
+
+g = lie.rand(5, lie.SE3)
+print("LieTensor is tensor", isinstance(g, torch.Tensor), "\n")
+
+# Can retract standard torch tensors
+x = torch.rand(5, 6)
+z = g.retract(x)
+
+# However, for safety, one cannot use overriden + with torch tensors
+try:
+    y = g + x
```

Comment:
Honestly, I'm a little nervous about overriding `__add()` since they are very dangerous. Also, one can not use `*`, `-`, `/`, either in that case.

Reply:
Do we have some operators called `to_euclidean()` that makes a pytorch tensor copy (not reference) of current data.

Reply:
In summary, I would like exactly function names rather than overrides to avoid confusion. I encountered lots of problems to generalize the first-order optimizers on Lie groups.

Reply:
To be consistent, if `g + x = g.compose(x)`, then `g-x` should be `g.compose(x.inverse())`.